gpc-api-facade/src/main/java/uk/nhs/adaptors/pss/gpc/exception/FhirValidationException.java

```diff
@@ -0,0 +1,7 @@
+package uk.nhs.adaptors.pss.gpc.exception;
+
+public class FhirValidationException extends RuntimeException {
```

Comment:
At the moment this exception is only used when parsing incoming fhir resource - maybe we could get rid of it now and use the one that HAPI throws. Then - in the validation/error handling story - add custom exceptions if necessary.